.prettierignore

```diff
@@ -0,0 +1,15 @@
+*.*
+!*.js
+!*.css
+!*.scss
```

Comment:
Do we need to ignore `.css` and `.scss` files? I'm not aware of any in this repo.